src/main/kotlin/com/cognifide/gradle/aem/api/AemNotifier.kt

```diff
@@ -1,48 +1,69 @@
 package com.cognifide.gradle.aem.api
 
-import dorkbox.notify.Notify
-import org.apache.commons.lang3.StringUtils
+import fr.jcgay.notification.*
 import org.apache.commons.lang3.exception.ExceptionUtils
 import org.gradle.api.Project
-import javax.imageio.ImageIO
+import org.gradle.api.logging.LogLevel
+import java.util.concurrent.TimeUnit
 
 class AemNotifier private constructor(private val project: Project) {
 
-    private val config by lazy { AemConfig.of(project) }
+    enum class Level {
+        ERROR, WARNING, INFO
+    }
 
-    fun log(title: String, message: String) {
-        project.logger.lifecycle(if (message.isNotBlank()) {
+    private val config by lazy { AemConfig.of(project) }
+    private val icon by lazy { Icon.create(javaClass.getResource(IMAGE_PATH), "default") }
+    private val application by lazy {
+        Application.builder()
+                .id("gradle-aem-plugin")
+                .name("Gradle AEM Plugin")
+                .icon(icon)
+                .timeout(TimeUnit.SECONDS.toMillis(5))
+                .build()
+    }
+    private val levelToLog: HashMap<Level, LogLevel> = hashMapOf(
+            Level.ERROR to LogLevel.ERROR,
+            Level.WARNING to LogLevel.WARN,
+            Level.INFO to LogLevel.LIFECYCLE)
+    private val levelToNotify: HashMap<Level, Notification.Level> = hashMapOf(
+            Level.ERROR to Notification.Level.ERROR,
+            Level.WARNING to Notification.Level.WARNING,
+            Level.INFO to Notification.Level.INFO)
+
+    fun log(title: String, message: String, level: Level = Level.INFO) {
+        val logLevel = levelToLog.getOrDefault(level, LogLevel.LIFECYCLE)
+        project.logger.log(logLevel, if (message.isNotBlank()) {
             "${title.removeSuffix(".")}. $message"
         } else {
             title
         })
     }
 
-    fun default(title: String, message: String = "") {
+    fun default(title: String, message: String = "", level: Level = Level.INFO) {
         if (config.notificationEnabled) {
-            now(title, message)
+            notify(title, message, level)
         }
 
-        log(title, message)
-    }
-
-    // TODO allow to customize color of plugin logo (warn, info, error etc)
-    fun now(title: String, text: String = "") {
-        now {
-            title(title)
-            text(StringUtils.replace(text, "\n", "<br>"))
-        }
+        log(title, message, level)
     }
 
-    fun now(configurer: Notify.() -> Unit) {
+    fun notify(title: String, text: String = "", level: Level = Level.INFO) {
+        val notifier = SendNotification()
+                .setApplication(application)
+                .initNotifier()
+        val notification = Notification.builder()
+                .title(title)
+                .message(text)
+                .icon(icon)
+                .level(levelToNotify.getOrDefault(level, Notification.Level.INFO))
+                .build()
         try {
-            Notify.create()
```

Comment:
previous notifications was having nice animated line which was interesting. maybe we could let plugin users to choose which notifications should be displayed

```
aem {
    config {
        notificationStyle = props.string('aem.notification.style', 'dorkbox')
    }
}
```

the question is, if native or dorkbox's notification should be displayed by default?

last thing, the notification config lambda should take an object as arg which will will be once dorkbox's builder and another time jcgay's notification builder.